fsthttp/backend.go

```diff
@@ -27,6 +27,12 @@ type Backend struct {
 	target string
 }
 
+func NewBackend(name string) *Backend {
+	return &Backend{
+		name: name,
+	}
+}
```

Comment:
Is the use case here to instantiate a `Backend` for an existing static backend?  If so, what do you think about calling this something like `BackendForName`, checking `Exists()` in the constructor and returning an `ErrBackendNotFound` if it's not there?

Reply:
And furthermore, for static backends does it make sense to set `target` to the value of `BackendGetHost`?  I dunno how that's supposed to map, exactly.

Reply:
Good point, that probably makes more sense.